feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts

```diff
@@ -86,13 +85,19 @@ export class ConfiguratorGroupMenuComponent {
   ) {}
 
   /**
-   * Fired on key board events, checks for 'enter' and delegates to click.
+   * Fired on key board events, checks for 'enter' or 'space' and delegates to click.
    *
    * @param {KeyboardEvent} event - Keyboard event
    * @param {Configurator.Group} group - Entered group
```

Comment:
As far as I see this comment does not belong to this method. Move the comment to the right method, namely **clickOnEnter**

Reply:
Done